File: Models/TaskModel.cs
using System;

namespace Models
{
    public class TaskModel
    {
        public int TaskModelId { get; set; }
        public string Title { get; set; }
        public string Body { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int TaskStatusModelId { get; set; }
    }
}

File: AngularApp/ClientApp/src/app/tasks/taskModel.ts
import { TaskStatusModel } from "./taskStatusModel";

export interface TaskModel {
  taskModelId: number;
  title: string;
  body: string;
  createdDate: Date;
  startDate: Date;
  endDate: Date;
  userId: number;
  taskStatusModelId: number;
  taskStatusModels: TaskStatusModel[];
}
